lib/rubygems/test_case.rb

```diff
@@ -11,11 +11,6 @@
   Gem::Specification.dirs.shift
 end
 
-begin
-  gem 'minitest', '~> 5.0'
```

Comment:
Are we suppose to be using `bundle exec rake test` from now on? I'm lost why we don't need this anymore

Reply:
I can see how you're lost because removing this is just wrong :laughing:. Nice catch, removing the commit.

Reply:
@colby-swandale I ended up restoring this but removing the exception swallowing code. Since after these changes our test suite requires minitest `5.13` or higher because I removed the compatibility code, I think the exception is useful to tell users not satisfying the requirement to install a matching version. Let me know what you think.

Reply:
That sounds reasonable. Would it be worth catching the exception anyway and add a friendly error message to say you need to run `bundle install`?

Reply:
@colby-swandale Since our tests are run under different environments, I think I'd rather the raw exception in place? For example, if our tests are run from `ruby-core`, I don't think a `bundle install` suggestion would apply there. Plus, `gem install minitest` will fix the issue anyways without a full `bundle install`.